tests/cc/current_py_cc_libs/BUILD.bazel

```diff
@@ -31,3 +31,19 @@ cc_test(
         "@rules_python//python/cc:current_py_cc_libs",
     ],
 )
+
+# This is technically a headers test, but since the pyconfig.h header
+# designates the appropriate lib to link on Win+MSVC, this test verifies that
+# the expected Windows libraries are all present in the expected location.
+# Since we define the Py_LIMITED_API macro, we expect the linker to go search
+# for libs/python3.lib.
+# buildifier: disable=native-cc
+cc_test(
+    name = "python_libs_linking_windows_test",
```

Comment:
CI is failing saying PyRun_SimpleString isn't defined. I don't think it's part of the limited API: https://docs.python.org/3/c-api/stable.html#contents-of-limited-api

It doesn't look like _any_ `PyRun_*` APIs are part of limited :unamused: . Looking at https://docs.python.org/3/c-api/veryhigh.html, I think what you'd have to do for an equivalent is `Py_Compile()` and then `PyEval_EvalCode()`.

However, since we're just checking linking, you could just delete that `PyRun` call. The code isn't runnable today anyways, and the other symbols should suffice to verify they can be resolved.